config/initializers/govuk_design_system_formbuilder.rb

```diff
@@ -1,5 +1,5 @@
 GOVUKDesignSystemFormBuilder.configure do |conf|
-  conf.default_error_summary_title  = 'Please correct the following errors'
+  conf.default_error_summary_title  = 'There is a problem'
```

Comment:
I think `There is a problem` is a placeholder in the designs. I'd be hesitant changing this and want to be absolutely confident this is what Content wants

Reply:
And even if this is something that UX/Content want, can we raise a separate PR for it? Scope creep = PR bloat = hard to review PRs

Reply:
Confirmed with content. I can do a separate PR.
For my own reference: `f0be6f1509aa9d80ae0ab44cc85ef65264959cf9` is the main commit.

Reply:
https://github.com/DFE-Digital/teacher-vacancy-service/pull/1771